ReleaseProcessAutomation/Program.cs

```diff
@@ -39,6 +40,11 @@ public static int Main (string[] args)
     {
       app.Run(args);
     }
+    catch (UserDoesNotWantToContinueException u)
+    {
+      Console.WriteException(u, ExceptionFormats.ShortenEverything);
+      return -1;
+    }
```

Comment:
Do we want to print out a user abortion as exception? The message is always going to be empty. Maybe we just print the message as an info and supply a "User aborted the xxx operation" when throwing the exception

Reply:
Ah yeah I did think of that when writing that, but I had the plan of reusing the exception, but that would probably be better done in another ticket. Will create one.

Reply:
Nevermind, I can't read. Thought of something else, will fix this and create a ticket for the other thing.